spec/system/candidate_interface/references/candidate_adding_referees_in_sandbox_spec.rb

```diff
@@ -57,4 +60,21 @@ def then_i_see_that_references_are_given
       expect(all('.govuk-summary-list__value')[4].text).to have_content('Reference given')
     end
   end
+
```

Comment:
I think these steps warrant their own system spec file, leaving this one to focus on the sandbox behaviour.